storage/replica_command.go

```diff
@@ -1648,7 +1648,7 @@ func (r *Replica) applyNewLeaseLocked(
 			// in on the Lease struct.
```

Comment:
`StartStasis` is when the lease expires (and then it's unusable during the stasis period, which lasts until `Expiration`).